jablib/build.gradle.kts

```diff
@@ -181,6 +181,33 @@ dependencies {
 
     implementation("de.rototor.snuggletex:snuggletex-jeuclid")
 
+    // region for document importing
```

Comment:
Please rework with the new scheme.

But the concrete versions at https://github.com/JabRef/jabref/blob/main/versions/build.gradle.kts